resources/config_conformance_pack.go

```diff
@@ -85,10 +88,17 @@ func ConfigConformancePack() *schema.Table {
 func fetchConfigConformancePacks(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
 	c := meta.(*client.Client)
 	config := configservice.DescribeConformancePacksInput{}
+	var ae smithy.APIError
 	for {
 		resp, err := c.Services().ConfigService.DescribeConformancePacks(ctx, &config, func(options *configservice.Options) {
 			options.Region = c.Region
 		})
+
+		// This is a workaround untill this bug is fixed = https://github.com/aws/aws-sdk-go-v2/issues/1539
+		if c.Region == "af-south-1" && errors.As(err, &ae) && ae.ErrorCode() == "AccessDeniedException" {
```

Comment:
Does it happen only in af-south region? Should we enable all the regions and test it.

Reply:
I enabled all of them and it happens only in af-south-1 even if it enabled. this is why I opened the issue in aws github